### DISCUSSION TITLE
docker部署

### DISCUSSION THREAD

**Discussion by Duan-0916** [vesoft-inc/nebula#5682] [General]
在使用docker pull拉取下来Nebula Graph镜像后使用./nebula-console -addr <ip> -port <port> -u <username> -p <password>这个命令去连接的时候出现nebula-console 找不到的情况，这种情况怎么解决

**Comment by wey-gu** [comment]
抱歉，这是因为 compose 之前的版本里 console 的 tag 写错了，昨天已经修复了，可以参考 https://github.com/vesoft-inc/nebula-docker-compose/pull/151/files 修改一下 yaml 再执行 docker-compose up -d 哈

**Reply by Duan-0916** [reply]
只能使用docker-compose执行操作数据库么，我需要使用docker镜像在容器里去操作数据库

**Reply by wey-gu** [reply]
这个就是更新 console 的容器镜像哈，你是说 console 必须在容器里么？不是必须的，可以直接下载二进制包在容器外使用 https://github.com/vesoft-inc/nebula-console/releases

**Reply by Duan-0916** [reply]
好的，谢谢

**Reply by Duan-0916** [reply]
下载完二进制包应该怎么用呀

**Reply by wey-gu** [reply]
https://docs.nebula-graph.com.cn/3.6.0/nebula-console/

参考文档，一般来说文件要给增加可执行权限比如 `chmod +x <name_of_console>`
